org.osgi.test.common/src/main/java/org/osgi/test/common/context/CloseableBundleContext.java

```diff
@@ -15,6 +14,10 @@
  * limitations under the License.
  */
 
+package org.osgi.test.common.context;
+
+import static aQute.lib.exceptions.Exceptions.duck;
```

Comment:
I think we should define our own duck method in commons rather than conditional packaging from Bnd. We don't know what conflicts may arise in the future.

Reply:
It's not just duck, it's all the other goodness in Exceptions that I envision using, *WithExceptions for instance, and `Exceptions.unchecked` etc...

Reply:
So copy or shade everything we actually need. The jars we build are not necessarily used as OSGi bundles, so we should be careful to avoid our copy of the package conflicting with someone elses.